.github/ISSUE_TEMPLATE/01_bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
```

Comment:
What is the reason you suggest to have this table at the top of the issue/PR? It confuses me a little, in particular because the "about" is also clarified at "bug details", "name" is similar to "labels", and "assignees" is usually chosen manually. What am I missing here?